R/writeModel.R

```diff
@@ -25,62 +23,71 @@
 ##' removed before the BUGS code is saved.
 ##' In S-PLUS, a warning is generated when the model function is defined if the
 ##' last statement in the model is an assignment. To avoid this warning, add the
-##' line \code{invisible()} to the end of the model definition. This line will be
-##' removed before the BUGS code is saved.
+##' line \code{invisible()} to the end of the model definition. This line will
+##' be removed before the BUGS code is saved.
 ##'
-##' @param model R function containing the BUGS model in the BUGS
-##' model language, for minor differences see Section Details.
+
+#' @param model R function containing the JAGS model
+#'  in the BUGS ##' model language, for minor differences see Section Details.
 ##' @param con passed to \code{\link{writeLines}} which actually writes the
 ##' model file
 ##' @param digits number of significant digits used for \pkg{WinBUGS}
```

Comment:
I would rather word it for usage with JAGS instead of WinBUGS since we don't support the latter